core/controllers/suggestion.py

```diff
@@ -470,13 +470,14 @@ def _construct_exploration_suggestions(suggestions):
         exploration_content_html field representing the target
         exploration's current content.
     """
+    exp_ids = {suggestion.target_id for suggestion in suggestions}
+    exp_id_to_exp = exp_fetchers.get_multiple_explorations_by_id(list(exp_ids))
+
     suggestion_dicts = []
     for suggestion in suggestions:
-        exploration = exp_fetchers.get_exploration_by_id(
-            suggestion.target_id)
-        content_html = exploration.get_content_html(
+        content_html = exp_id_to_exp[suggestion.target_id].get_content_html(
             suggestion.change.state_name, suggestion.change.content_id)
-        suggestion_dict = suggestion.to_dict()
+        suggestion_dict = suggestion.change.to_dict()
```

Comment:
@vojtechjelinek -- this seems wrong. Why did "suggestion" become "suggestion.change"?

Reply:
Also it's worrying that this wasn't caught by tests, may be worth adding some.